doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -451,6 +451,7 @@ Bluetooth Fast Pair samples
     * The location of the sample configuration.
       It has been moved from the root sample directory to the dedicated folder (:file:`locator_tag/configuration`).
     * The ``fp_adv`` module to use the trigger requests for the Fast Pair advertising state instead of setting the Fast Pair advertising mode directly.
+    * The memory map for the :ref:`zephyr:nrf54l15dk_nrf54l15` board to provide more space for the MCUboot bootloader.
```

Comment:
DFU and partition maps were added during this release so no need for the changelog entry.

Reply:
Probably yes, as it seems not needed

Reply:
yes, removed